actions.yaml

```diff
@@ -7,6 +7,14 @@ synchronize:
     and synchronize the package metadata (index files) with the upstream. In
     addition, unreferenced packages will be automatically cleaned up without
     confirmation; also see check-packages and clean-up-packages actions.
+  params:
+    source:
+      description: |
```

Comment:
Can you provide an example here

Reply:
I'll :+1:

Reply:
If I run
`juju run-action apt-mirror/0 -- synchronize source=".*security.*"`
It will sync everything than contains `security` in it and it will delete everything else.

Reply:
Also can you update the README.md with a few examples.

Reply:
What do you mean it remove everything else?

Reply:
I feel there is an ambiguity in the wording `In addition, unreferenced packages will be automatically cleaned up without confirmation; also see check-packages and clean-up-packages actions.` However it not releated with this commit.